src/components/ConversationSettings/BasicSettings.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+	<Fragment>
+		<NcTextField :value.sync="conversationNameTextField"
+			:label="t('spreed', 'Name')"
+			:disabled="!isEditing"
+			:label-visible="true" />
+		<NcTextField :value.sync="descriptionTextField"
```

Comment:
This used to be a `RichContentEditable` which brings mentions and emoji autocomplete.
I think we should have that if possible.